Braintree.xcodeproj/project.pbxproj

```diff
@@ -2967,6 +2970,7 @@
 			buildActionMask = 2147483647;
 			files = (
 				80B59A132B27C7FC004C2FA3 /* BTShopperInsightsClient_Tests.swift in Sources */,
+				62970D102B5AF2C000BAB584 /* BTShopperInsightsAnalytics_Tests.swift in Sources */,
```

Comment:
Super nitpick, but we try to always sort the files by name. Can we bump this above the client tests? Should be able to right click the directory then click "sort by name"

Reply:
yep absolutely, addressed in 5c16b60b274c998158563c2f6a4080e2aa991891